src/main/java/org/gridsuite/explore/server/services/ContingencyListService.java

```diff
@@ -61,12 +61,12 @@ public void insertScriptContingencyList(UUID id, String content) {
         restTemplate.exchange(actionsServerBaseUri + path, HttpMethod.POST, httpEntity, Void.class);
     }
 
-    public void insertScriptContingencyList(UUID sourceListId, UUID id) {
+    public UUID duplicateScriptContingencyList(UUID scriptContingencyListsId) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + ACTIONS_API_VERSION + "/script-contingency-lists")
-                .queryParam(HEADER_DUPLICATE_FROM, sourceListId)
-                .queryParam("id", id)
+                .queryParam(HEADER_DUPLICATE_FROM, scriptContingencyListsId)
+                .buildAndExpand(scriptContingencyListsId)
```

Comment:
to remove ther isn't any {id} in the path to replace

Reply:
done